README.rst

```diff
@@ -2,25 +2,53 @@
 Purpose of STScI-MIRI Space
 ###########################
 
-The goal of this github is to have a central location to store
+The goal of this github organization is to have a central location to store
 repositories of code used by the STScI MIRI team as part of
 carrying out their functional MIRI related duties.
 
 Having a central location for such code provides for long-term
-curation of the code.  In addition it encourages more efficent development
+curation, backup, and team ownership of the code.  In addition it encourages more efficent development
 of code by taking advantage of the expertise of multiple team members.
 
+STScI-MIRI is classified as an educational organization, and therefore
+can create unlimited private repositories.
+
 Space Organization
 ==================
 
 A subset of the team members have the responsiblity to manage
 the space and provide guidence and training to all team members
 on effective use of this space, git/github, and
-distributed team-based code development.  
+distributed team-based code development.  Space ownership is divided between two or
+more MIRI team members to ensure administrative support in the event
+of one person being unavailable.
+
+All members are required to use two-factor authentication when working
+with repositories within the STScI-MIRI space.
+
+Linked Data Directories
+==================
```

Comment:
This line needs to be just as long as the title above.  Requirement of rst format.